File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.java
class Solution {
    public int[] solution(int[] numbers, int num1, int num2) {
        int a = 0;
            a = num2 - num1 + 1;
        
        int[] answer = new int[a];
        int count = 0;
        
        for (int i = num1; i <=num2; i++) {
            answer[count] = numbers[i];
            count++;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120833. 배열 자르기/배열 자르기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int numbers[], size_t numbers_len, int num1, int num2) {
    int* answer = (int*)malloc(sizeof(int)*num2);
    int count = 0; 
    for (int i = num1; i <= num2; i++) {
        answer[count] = numbers[i];
        count++;
    }
    return answer;
}